src/kagglehub/auth.py

```diff
@@ -33,7 +33,6 @@ def _capture_logger_output():
     """
     buffer = io.StringIO()
     handler = logging.StreamHandler(buffer)
-    logger = logging.getLogger()
```

Comment:
This was capturing the output from the root logger. However, now that propagation is disabled from `kagglehub`, we should make sure we use a logger under `kagglehub`.